src/cppcache/src/CacheXmlParser.cpp

```diff
@@ -1165,7 +1166,7 @@ void CacheXmlParser::startPersistenceManager(const xmlChar** atts) {
         throw CacheXmlException(s.c_str());
       }
 
-      ACE_OS::strncpy(libraryFunctionName, (char*)atts[i], len);
+      std::strncpy(libraryFunctionName, (char*)atts[i], len);
```

Comment:
Maybe not is a great opportunity to fix these c-style casts that are warnings in C++11 now when you are fixing other things in the file.